src/mumble/AudioOutput.cpp

```diff
@@ -102,9 +102,11 @@ float AudioOutput::calcGain(float dotproduct, float distance) {
 	float dotfactor = (dotproduct + 1.0f) / 2.0f;
 	float att;
 
-
-	// No distance attenuation
-	if (Global::get().s.fAudioMaxDistVolume > 0.99f) {
+	if (distance < 0.0001f) {
+		// Listener is inside source -> no attenuation
+		att = 1.0f;
+	} else if (Global::get().s.fAudioMaxDistVolume > 0.99f) {
+		// No distance attenuation
 		att = qMin(1.0f, dotfactor + Global::get().s.fAudioBloom);
```

Comment:
While touching this part of the code, we can also replace `qMin` with `std::min` in `calcGain` (there's another use of `qMin` a few lines down)